### PR TITLE
rtss: Update to version 7.3.1

### PR DESCRIPTION
**Pull request by stasjok**
There is an OverlayEditor plugin in version 7.3.0 which saves overlays to `Plugins\Client\Overlays\` (even if you choose another directory). I've added it to persist.
Checkver was disabled in commit https://github.com/lukesampson/scoop-extras/commit/d1fa5192c3268bd9e3d130413de0521d0a532a0c. I didn't touch it.

### CODE REVIEW

**Comment by Fooughhy** [comment]
This update also fixes an issue with an annoying popup I get during boot, "Some system components cannot be hooked right now" which I believe simply was an incompatability issue with an update to Windows 10.

Would appreciate if this could be merged 😄

File: bucket/rtss.json [hunk 1]
```diff
@@ -31,8 +31,12 @@
             "RivaTuner Statistics Server"
         ]
     ],
-    "persist": "Profiles",
-    "autoupdate": {
```
Comment (pratikpc) [inline bucket/rtss.json:35]: Why did you remove autoupdate?
